gapic/packaging/api_defaults.yaml

```diff
@@ -16,14 +16,18 @@
 # Provides default values used as metadata when constructing various packages.
 
 author: Google Inc
-description: a grpc-based api
 email: googleapis-packages@google.com
-github_user_uri: https://github.com/googleapis
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
-semver:
+package_version:
```

Comment:
Hmm, maybe `this_package_version` or `generated_package_version` then? I think just `package_version` is a context-less description and it seems like it could apply to anything.